### ISSUE TITLE
File Uploads?

### ISSUE THREAD

**Issue by ghost** [evancz/elm-http#43]
Any Idea on how I'm going to do file uploads. I was searching the source and could only find multipart string upload? Will that be useful?

**Comment by magopian** [comment]
TL;DR for maintainers: As issues are for bugs, and this doesn't seem like a bug, I believe it can be closed.

I'm pretty sure questions are supposed to be asked in one of the community forums (check https://github.com/process-bot/contribution-checklist/blob/master/issues.md#issues).

I've seen a couple of solutions by doing a quick google search
- using the [file-reader lib](https://github.com/simonh1000/file-reader): http://stackoverflow.com/a/35393219
- using ports: https://gist.github.com/mmport80/68d142e346cf906f4d9d

If you feel that there's a need for better documentation, or for the implementation of such a functionality in the elm-http package, or even just an entry in the elm FAQ (would it be the correct place for it?), please open an issue for that matter.

Thanks!

**Comment by ghost** [comment]
Hmm.. Thanks I couldn't find that after searching on github since I thought this was impossible to do in elm and ditched the effort to go further because of this, now I can start playing with it again.
